overlays/musl.nix

```diff
@@ -9,6 +9,7 @@ final: prev: prev.lib.optionalAttrs prev.stdenv.hostPlatform.isMusl ({
   zlib = prev.zlib.override { splitStaticOutput = false; };
 
   # and a few more packages that need their static libs explicitly enabled
+  bzip2 = prev.bzip2.override { enableStatic = true; };
```

Comment:
now it's in both?

Reply:
Well `//` would have replaced `bzip2`, but I have refactored it to make it clearer and added a comment.